hack/verify-api-groups.sh

```diff
@@ -111,12 +111,10 @@ for expected_install_package in "${expected_install_packages[@]}"; do
 		continue
 	fi
 
-	for known_version_file in "${known_version_files[@]}"; do
```

Comment:
```
In ./hack/verify-api-groups.sh line 114:
        for known_version_file in "${known_version_files[@]}"; do
        ^-^ SC2034: known_version_file appears unused. Verify use (or export if used externally).
```
known_version_file has been removed because it is not used.